lib/vet360/models/base.rb

```diff
@@ -11,6 +11,26 @@ class Base
       include Virtus.model(nullify_blank: true)
 
       SOURCE_SYSTEM = 'VETSGOV'
+      DEFAULT_ATTRS = %i[effective_start_date source_date vet360_id].freeze
+
+      def self.with_defaults(user, params)
+        instance = new(params)
+        now      = Time.now.iso8601
+
+        return unless instance.default_attrs_present?
```

Comment:
This guard should raise an exception rather than returning nil.

Based on discussion, this exception may be suitable: https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/common/exceptions/internal/internal_server_error.rb